data/sitesEN.json

```diff
@@ -1701,14 +1701,20 @@
   },
   {
     "id": "en-doctorwho",
-    "origins_label": "Tardis Fandom & Doctor Who Neoseeker Wikis",
+    "origins_label": "Whoniverse Fandom & Neoseeker Wikis",
```

Comment:
What do you think of setting the label to "Doctor Who" instead of "Whoniverse"?

Reply:
A bit inaccurate, as :
- "Whoniverse" is a term to refer to the entire Doctor Who franchise. 
-  TARDIS wiki covers all Doctor Who content produced by the BBC regardless of canonicity (as much as Doctor Who can have a canon) as demonstrated by this page for a [spoof Seventh Doctor](https://tardis.wiki/wiki/Seventh_Doctor_(The_Lenny_Henry_Show))